integration_tests/lxp-bridge.config.yaml

```diff
@@ -0,0 +1,40 @@
+inverters:
+- enabled: true
+  host: 127.0.0.1
+  port: 8000
+  serial: 5555555555
+  datalog: 2222222222
+  heartbeats: true
+  # FIXME: get the specs passing with this enabled
```

Comment:
This isn't actually in `master` yet.

Since we're now running an lxp-bridge per spec, we can vary the config-file per spec, rather than having it hardcoded. That's a few revisions away though.